docker/driver.go

```diff
@@ -0,0 +1,96 @@
+package docker
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/bblfsh/performance"
+
+	errors "gopkg.in/src-d/go-errors.v1"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// One day it will become a config file but now it's to annoying to ship it
+var drivers = map[string]string{
+	"bash":       "bash-driver",
+	"cpp":        "cpp-driver",
+	"csharp":     "csharp-driver",
+	"go":         "go-driver",
+	"java":       "java-driver",
+	"javascript": "javascript-driver",
+	"php":        "php-driver",
+	"python":     "python-driver",
+	"ruby":       "ruby-driver",
+	"typescript": "typescript-driver",
+}
+
+// InstallDriver downloads driver's repository, checkouts to a given commit,
+// builds docker image and installs driver's image to bblfshd container.
+// Requires bblfshd container running.
+func InstallDriver(language, commit string) error {
+	if err := validateLanguage(language); err != nil {
```

Comment:
With the suggestion from line 14 this could be:

```go
driver, err := driverName(language)
if err != nil {
  return err
}
// …do stuff with `driver`…
```

Reply:
Done